projects/plugins/jetpack/modules/masterbar/inline-help/inline-help.css

```diff
@@ -0,0 +1,68 @@
+.inline-help {
+  position: fixed;
```

Comment:
You'll want to use tabs instead of spaces in all that file, to avoid folks accidentally editing all lines when they'll edit this file in their editor in the future.

Reply:
I've used my editor to convert to tabs. Looks better?